### PR TITLE
feat: inserting imgs to hero

### PR DESCRIPTION
**Pull request by assebc**
no prints please open and test

## Type of change

Please delete options that are not relevant or select with a "x" the correct option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist(mark with a "x" the items you made):

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I tested all related functionalities and I didn´t break anything

### CODE REVIEW

File: src/components/molecules/Hero/ContactUsHero/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    padding: 4% 20%;
+
+    gap: 48px;
+
+    animation: fadeIn 5s;
+}
+
+    .contact-us-hero-wrapper video {
+        width: 100%;
+        height: 31%;
```
Comment (Mariii-23) [inline src/components/molecules/Hero/ContactUsHero/styles.css:20]: /4

File: src/components/molecules/Hero/ContactUsHero/index.tsx
[the remark this answers is not in the document]
Comment (assebc) [reply src/components/molecules/Hero/ContactUsHero/index.tsx:1]: which whitespace?

**Review by Mariii-23 — CHANGES_REQUESTED** [review]
Resolve the conflicts pls

**Review by Mariii-23 — APPROVED** [review]
I think it's important that we leave some pages without the videos so we don't get so tired and get more impact. Pages like the contactUs and the services.

LGTM

**Pull request merged**